src/stores/networkStore.ts

```diff
@@ -0,0 +1,77 @@
+import { action, computed, observable } from 'mobx';
+import TransitType from '../enums/transitType';
+
+export class NetworkStore {
+    @observable private _enabledTypes: TransitType[];
+    @observable private _showLinks: boolean;
+    @observable private _showNodes: boolean;
+    @observable private _showPoints: boolean;
+
+    constructor() {
+        this._enabledTypes = [
+            TransitType.BUS,
+            TransitType.FERRY,
+            TransitType.SUBWAY,
+            TransitType.TRAIN,
+            TransitType.TRAM,
+        ];
+        this._showLinks = false;
+        this._showNodes = false;
+        this._showPoints = false;
+    }
+
+    private isAnythingEnabled() {
+        return this._showLinks || this._showNodes;
+    }
+
+    @computed get showLinks(): boolean {
```

Comment:
Generally a verb is a bad name for a boolean. "isLinksVisible / isLinksShown" would be a better name